tmt/schemas/stories.yaml

```diff
@@ -0,0 +1,62 @@
+---
+
+#
+# JSON Schema definition for tmt `Stories`
+#
+# https://tmt.readthedocs.io/en/stable/spec/stories.html
+#
+
+$id: /schemas/stories
+$schema: https://json-schema.org/draft-07/schema
+
+type: object
+additionalProperties: false
+
+properties:
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#adjust
+  adjust:
+    $ref: "/schemas/core#/definitions/adjust"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#description
+  description:
+    $ref: "/schemas/core#/definitions/description"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#enabled
+  enabled:
+    $ref: "/schemas/core#/definitions/enabled"
+
+  # https://tmt.readthedocs.io/en/stable/spec/stories.html#example
+  example:
+    $ref: "/schemas/common#/definitions/string_or_array"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#link
+  link:
+    $ref: "/schemas/core#/definitions/link"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#order
+  order:
+    $ref: "/schemas/core#/definitions/order"
+
+  # https://tmt.readthedocs.io/en/stable/spec/stories.html#priority
+  # TODO: missing from documentation
```

Comment:
FTR, https://github.com/teemtee/tmt/pull/440 adds the specification.

Reply:
Unfortunately, #440 will need more discussion, the meeting is scheduled for Thursday, I would say we will need to update this once we are clear on #440.

Reply:
```suggestion
# TODO: might need update once https://github.com/teemtee/tmt/pull/440 is merged
```